### DISCUSSION TITLE
Nonmodular stations

### DISCUSSION THREAD

**Discussion by PrismMind** [FakeFishGames/Barotrauma#5781] [Ideas and suggestions]
The stations and outposts in Barotrauma are built out of a set of modular parts that are semi randomized to give stations a different layout from one another. I often find this makes stations rather dull but more importantly its very difficult to add new parts for stations with out accidently breaking things having the "tone" of the station all mismatched. I'm suggesting a option for the sub editor to save a sub file as a full station where no parts will be attempted to be added on by the game this way more attention to large scale detail can be made by sub builders and it would be far easier to upload custom stations onto the workshop that would appear in game.

**Comment by Cogs16** [comment]
I think additionally, all non-wrecked and wrecked variants of parts are somehow tagged or linked, so that things can be automatically converted between wrecked and non wrecked versions.
